snap/snapcraft.yaml

```diff
@@ -146,6 +146,16 @@ parts:
     prime:
       - -htdocs/apps/updatenotification
 
+  # libsmbclient is needed by the SMB PHP module, but it depends upon python.
+  # Pulling in python via stage-packages conflicts with the python from the
+  # python plugin used in certbot, so we'll use the python plugin too to
+  # install libsmbclient.
+  # See: https://github.com/nextcloud/nextcloud-snap/pull/263
```

Comment:
Oops, my comment got eaten by the diff. Can you please remove the link to the PR? It's unnecessary now as the comment explains everything. If people want to know where the change came from, they can use git (and find the PR from there).